### PR TITLE
Builder

### PR DESCRIPTION
**Pull request by lgywata**
Remove unused code and change a few parameters names for builder API.

### CODE REVIEW

File: src/lib/flow/sol-flow-builder.h [hunk 1]
```diff
@@ -69,15 +69,10 @@ int sol_flow_builder_set_type_description(struct sol_flow_builder *builder, cons
...
 int sol_flow_builder_add_node(struct sol_flow_builder *builder, const char *name, const struct sol_flow_node_type *type, const struct sol_flow_node_options *option);
 
-/* Add nodes to nodes spec by id. It'll use a resolver to get
- * node type and options.
- * Node ids can't be NULL and must to be unique */
-int sol_flow_builder_add_node_by_id(struct sol_flow_builder *builder, const char *id);
-
 /* Add nodes to nodes spec by its type name. It'll use a resolver to get
  * node type and options.
  * Node ids can't be NULL and must to be unique */
```
Comment (cmarcelo) [inline src/lib/flow/sol-flow-builder.h]: "Node ids can't be..." should be "Node names can't be..."

**Comment by cmarcelo** [comment]
@ibriano

+1 just fix the comment.

File: src/lib/flow/sol-flow-builder.h (hunk 1 above)
Comment (ibriano) [inline src/lib/flow/sol-flow-builder.h]: But that code is being deleted, nothing to fix there.

Comment (ibriano) [inline src/lib/flow/sol-flow-builder.h]: Oh, the one below.

**Comment by ibriano** [comment]
+1

**Comment by lgywata** [comment]
Fixed and applied. Thanks.

**Pull request closed without merging**